src/Templates/DiffPlaces/templateGen.js

```diff
@@ -679,7 +681,7 @@ ${testName}:
         value: !!int -1
 
       network:
-        - '>=London'
+        - '${fork}'
```

Comment:
if yaml can fit here fork into list  
```
- Berlin
- Merge
- Shanghai
```

would be nice

Reply:
Ask and [you shall receive](https://github.com/ethereum/tests/blob/9979ea5d63f1e43221e74f7f7b255f415655aaa3/src/Templates/DiffPlaces/createDiffPlacesTests.sh#L8).